src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/HealthCheckSpecification.groovy

```diff
@@ -10,8 +10,13 @@ import org.openkilda.messaging.info.event.IslChangeType
 import org.openkilda.testing.model.topology.TopologyDefinition.Status
 import org.openkilda.testing.tools.SoftAssertions
 
+import org.springframework.beans.factory.annotation.Value
+
 class HealthCheckSpecification extends BaseSpecification {
 
+    @Value('${use.multitable}')
+    boolean useMultitable
```

Comment:
How about moving it to the `BaseSpecification` file?
After that, we will refactor other specs and delete defining of the `useMultitable` variable

Reply:
done, also removed duplicates from child specs